wordpress/wp-content/plugins/memberful-wp/src/private_user_feed.php

```diff
@@ -104,6 +101,9 @@ function memberful_private_rss_feed_link($success_message = '', $error_message =
 
   $link = (get_home_url() . '/' . memberful_private_user_feed_get_url_identifier($feedToken) );
 
+  if($category)
+    $link .= '&category=' . $category;
```

Comment:
I would like to suggest to use [`add_query_arg`](https://developer.wordpress.org/reference/functions/add_query_arg/) for this. It probably doesn't matter in this case, but it is a WordPress convention for adding parameters to URLs.